fr/mail/chrome/messenger/migration/migration.dtd

```diff
@@ -16,14 +16,14 @@
 <!ENTITY importFromNothing.label        "Ne rien importer">
 <!ENTITY importFromNothing.accesskey    "N">
 
-<!ENTITY welcome.title                  "Welcome to &brandShortName;">
+<!ENTITY welcome.title                  "Bienvenue &brandShortName;">
 <!ENTITY tagline.label                  "A Better Way To Do Email">
```

Comment:
Sherman m'a demandé de traduire par une phrase en français qui correspondrait à "Awesome Email" ou "The power email"... J'ai mis "L'Email rendu incroyable", je modifierais si tu as une proposition autre